client-app/ui-kit/components/organisms/modal/vc-modal.vue

```diff
@@ -10,7 +10,7 @@
         leave-from="opacity-100"
         leave-to="opacity-0"
       >
-        <DialogOverlay class="fixed inset-0 z-50 bg-neutral-900 opacity-30" />
```

Comment:
it's deprecated [headlessui.com/v1/vue/dialog#dialog-overlay](https://headlessui.com/v1/vue/dialog#dialog-overlay)